src/Chapter3.hs

```diff
@@ -971,6 +1095,30 @@ implement the following functions:
 🕯 HINT: to implement this task, derive some standard typeclasses
 -}
 
+data WeekDay 
+    = Monday
+    | Tuesday
+    | Wednesday
+    | Thursday
+    | Friday
+    | Saturday
+    | Sunday
+    deriving (Show, Eq, Bounded, Enum)
+
+isWeekend :: WeekDay -> Bool
+isWeekend x
+      | (x == Saturday) || (x == Sunday) = True
+      | otherwise = False
```

Comment:
Great!
Actually, you can inline that without using guards, as your checks already return the answer 🙂 

```suggestion
isWeekend x = x == Saturday || x == Sunday
```